src/crypto/dleq.rs

```diff
@@ -958,79 +862,83 @@ impl DLEQProof {
     }
 }
 
-#[test]
```

Comment:
Don't bother reviewing beyond this point - it's just moved into tests module and the diff calc is a total mess.